### PR TITLE
I hate radiation and stupid things, new admin button to fix it and cleanable overhaul

### LINKED ISSUE #10540: FUCK Nuclear Wraiths
Today I found out if you sit in a radioactive cloud like the turbine while intangible to absorb radiation you can poison people without ever revealing yourself.

This is fucked up beyond belief. Wraiths can kill people without ever being vulnerable while people stand around and die miserably of radiation poisoning for no apparent reason. Fuck this bug make it burn.

### PR DESCRIPTION
**Pull request by JamieD1**
Overhauled radiation and cleanable systems

Ports: https://github.com/tgstation/tgstation/pull/51902
Ports: https://github.com/tgstation/tgstation/pull/52477
Ports: https://github.com/tgstation/tgstation/pull/52091

Closes: #10540 

#### Changelog

:cl:  
bugfix: Overhauled radiation and cleanable systems
/:cl:

### CODE REVIEW

**Review by redmoogle — CHANGES_REQUESTED** [review]
Remove the defines in datumvars or the ones in the other files

**Review by redmoogle — APPROVED** [review]
Code is tested and fine. Make sure to link all the issues(https://github.com/yogstation13/Yogstation/issues/2352)
and
https://github.com/yogstation13/Yogstation/issues/10540

File: code/__HELPERS/radiation.dm [hunk 1]
```diff
@@ -45,3 +45,14 @@
 		var/turf/_source_T = isturf(source) ? source : get_turf(source)
 		log_game("Radiation pulse with intensity: [intensity] and range modifier: [range_modifier] in [loc_name(_source_T)] ")
 	return TRUE
+
+/proc/get_rad_contamination(atom/location)
```
Comment (LemonInTheDark) [inline code/__HELPERS/radiation.dm:49]: When you port something https://github.com/tgstation/tgstation/pull/51902

File: code/__DEFINES/flags.dm [hunk 2]
```diff
@@ -45,6 +45,10 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define ADMIN_SPAWNED_1			    (1<<15)
 /// should not get harmed if this gets caught by an explosion?
 #define PREVENT_CONTENTS_EXPLOSION_1 (1<<16)
+/// should the contents of this atom be acted upon
+#define RAD_PROTECT_CONTENTS_1 (1 << 17)
+/// should this object be allowed to be contaminated
+#define RAD_NO_CONTAMINATE_1 (1 << 18)
```
Comment (LemonInTheDark) [inline code/__DEFINES/flags.dm:51]: Please credit the prs you're taking the code from https://github.com/tgstation/tgstation/pull/52091

File: code/game/turfs/open.dm [hunk 3]
```diff
@@ -331,10 +331,12 @@
 	for(var/mob/living/simple_animal/slime/M in src)
 		M.apply_water()
 
-	SEND_SIGNAL(src, COMSIG_COMPONENT_CLEAN_ACT, CLEAN_WEAK)
-	for(var/obj/effect/O in src)
-		if(is_cleanable(O))
-			qdel(O)
+	wash(CLEAN_WASH)
```
Comment (LemonInTheDark) [inline code/game/turfs/open.dm:334]: I don't intend to be rude I just want our coders to be credited https://github.com/tgstation/tgstation/pull/52477

**Pull request merged**